modules/util.py

```diff
@@ -531,13 +531,9 @@ def safeDirCopy(src, dst, sync=True, ignore=[]):
 
     # Copy each file in the structure of src to dst
     for root, dirs, files in os.walk(src):
-
             # Directories
+            dirs[:] = list(filter(lambda d: not d in ignore, dirs))
```

Comment:
NIT: this can also be a comprehension: `dirs[:] = [d for d in dirs if not d in ignore]`